go/kbfs/libkbfs/kbfs_ops.go

```diff
@@ -267,12 +267,62 @@ func (fs *KBFSOpsStandard) GetFavorites(ctx context.Context) (
 
 // GetFavoritesAll implements the KBFSOps interface for
 // KBFSOpsStandard.
-func (fs *KBFSOpsStandard) GetFavoritesAll(ctx context.Context) (keybase1.
-	FavoritesResult, error) {
+func (fs *KBFSOpsStandard) GetFavoritesAll(ctx context.Context) (
```

Comment:
Is the plan to use this for the SimpleFS interface for TLF metadata? I think we'll need some more than what's currently in `keybase1.Folder`: https://github.com/keybase/client/blob/dea76af9287733957df1f2123d605dd534d937d6/shared/constants/types/fs.js#L70-L89

I guess these could be filled by simpleFS though.

Reply:
I have no idea, I don't know where you currently get the TLF metadata.  Note that you don't just get a `Folder`, you get a `FavoriteResult`, which categorizes the favorite vs. new vs. ignored stuff already.  I don't know where this other stuff comes from.

Reply:
Oh that's right; that solves the ignored/new things. So that leaves us just the reset-related stuff I think. Currently they are from API server `kbfs/favorite/list`

Reply:
this interface is the same as the one provided by that API endpoint - the reset stuff is in `resetMembers` IIRC.

Reply:
Sweet; I'll look into plumbing this into GUI. I seem to have a de javu now and I think you've said that days ago. Sorry for forgetting that!